packages/xo-web/src/xo-app/xoa/support/index.js

```
@@ -0,0 +1,52 @@
+import _ from 'intl'
+import ActionButton from 'action-button'
+import AnsiUp from 'ansi_up'
+import decorate from 'apply-decorators'
+import React from 'react'
+import { adminOnly } from 'utils'
+import { Card, CardBlock, CardHeader } from 'card'
+import { Container, Row, Col } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { checkXoa } from 'xo'
+
+const ansiUp = new AnsiUp()
+
+const Support = decorate([
+  adminOnly,
+  provideState({
+    initialState: () => ({ stdoutCheckXoa: '' }),
+    effects: {
+      initialize: async () => ({ stdoutCheckXoa: await checkXoa() }),
```

Check that the `xoa` plugin is loaded, first.

---

Is it really necessary?

The plugin should always be installed and loaded on the XOA, I don't think we should really invest on checking this.

---

But a user can decide to unload it, right?

---

Yes (for now), but I don't think we should go to lengths to handle this because it's probably extremely rare and is not a normal condition.

We should just make sure there is no major issues in this case, but as long as XO is still usable I'm fine with this.

---

Ok sounds good. So we make sure to silently fail when the method doesn't exist (ie plugin not loaded/unloaded) but we don't add a specific warning message for this since it's not a normal case.